src/plugins/intel_cpu/tests/functional/single_layer_tests/classes/eltwise.cpp

```diff
@@ -168,6 +168,9 @@ const ov::AnyMap& additional_config() {
 
 const std::vector<ElementType>& netType() {
         static const std::vector<ElementType> netType = {
+#if defined(__ARM_FEATURE_FP16_VECTOR_ARITHMETIC)
```

Comment:
1. See how FP16 tests were added for Reduce https://github.com/openvinotoolkit/openvino/pull/18227 (via additional config with inference precision hint, not net precision)

Reply:
Don't we plan to support both cases (via whole model being f16 and via inference precision hint)?
It was the case for bf16 for example
Also I guess nothing stops user to create such model manually.

Reply:
In future it might be the case. For now only inference_precision hint is required. See this comment https://github.com/openvinotoolkit/openvino/pull/18394#discussion_r1274803427